[master<changelog-lab-260] Lab 2.6.0 changelog (#829)
memgraph/docs: lab/changelog.md

  TSV file format.
- If the dataset contains millions or billions nodes and relationships, their
  count in the status bar will be in the following format: X.XXM or X.XXB.
- Syntax of code blocks in the query collection description can now be
  highlighted by using one of the following language
  styles: `cypher`, `bash`, `python`, `css`, `c`, `cpp`, `json`, `sql`,
  and `yaml`. Check the examples of syntax highlighting in the [Markdown
  Guide](https://www.markdownguide.org/extended-syntax/#syntax-highlighting).
- New functions of the Graph Style Script language used for customizing graph
  appearance are: `Reduce`, `Sum`, `Avg`, `Min`, `Max`, `IsArray`, `Hue`,
  `Saturation`, `Lightness`, `HSL`.

### Bug fixes

- The initial node count has been removed from the connection initialization so
  connecting to the Memgraph instance containing a huge number of nodes will no
  longer cause a timeout. 
- Run History now logs queries as expected.
- When switching between the map view and the default view the graph view no
  longer becomes unresponsive. 
- Using a new line character `\n` in the query module doesn’t result in a new
  line, but an explicit character `\n`.

> - Viewing the code of multiple query modules in the split screen no longer now

Reviewer: ```suggestion
- Viewing the code of multiple query modules in the split screen now
```

  works as expected.
- Notifications no longer mix with the Query Editor and Query Collections visual
  elements.
- Using shortcut keys CMD/CTRL + S to save a query to a query collection now
  works for queries within a single query execution. In a query collection, the

>   shortcut will initiate a manual save of the query within the query collection.

Author: I don't really understand this one

> - Using shortcut keys CMD/CTRL + S to save a query to a query collection now
>   works for queries within a single query execution. In a query collection, the
>   shortcut will initiate a manual save of the query within the query collection.

Author: ```suggestion
- Pressing CMD/CTRL + S will save a query within a query collection execution section as intended.
```
